app/services/container_dashboard_service.rb

```diff
@@ -224,8 +224,8 @@ def fill_daily_pod_metrics(metrics, pod_create_trend, pod_delete_trend)
     metrics.each do |m|
       timestamp = m.timestamp.strftime("%Y-%m-%d")
 
-      pod_create_trend[timestamp] += m.stat_container_group_create_rate.to_i
-      pod_delete_trend[timestamp] += m.stat_container_group_delete_rate.to_i
+      pod_create_trend[timestamp] += m.stat_container_group_create_rate.to_i if m.stat_container_group_create_rate.present?
+      pod_delete_trend[timestamp] += m.stat_container_group_delete_rate.to_i if m.stat_container_group_delete_rate.present?
```

Comment:
@zeari you shouldn't need these since `to_i` is making sure that you have an integer here. (`nil` => 0)

Maybe you can adopt this for the rest as well (Be careful! some may be floats!).